crds/jwst/locate.py

```diff
@@ -181,7 +181,22 @@ def decompose_newstyle_name(filename):
 
     assert instrument in INSTRUMENTS+[""], "Invalid instrument " + repr(instrument)
     assert filekind in FILEKINDS+[""], "Invalid filekind " + repr(filekind)
+
+    # The original regular expresssion "\d*\" matches anything in the below assert statement, so it's basically a no-op.
```

Comment:
Suggest creating an issue now about this so it isn't forgotten.

Reply:
CCD-876 created.